File: 2022-06-craft-conf-languages/examples/fibonacci.rs
fn fib(x: u64) -> u64 {
    let mut a = 0;
    let mut b = 1;
    for _ in 0..x {
        let next = a + b;
        a = b;
        b = next;
    }
    a
}

fn main() {
    println!("{}", fib(10));
}

File: 2022-06-craft-conf-languages/examples/fibonacci.pl
sub fib {
    my $x = shift(@_);
    my $a = 0;
    my $b = 1;
    foreach (0..$x - 1) {
        my $next = $a + $b;
        $a = $b;
        $b = $next;
    }
    return $a;
}

print(fib(10));
print "\n"
